### ISSUE TITLE
Attempts to upsell but pro monitoring availability code seems to be missing

### ISSUE THREAD

**Issue by AndrewX192** [arcus-smart-home/arcusweb#2]
When creating an account I'm prompted as to whether I want to signup for pro monitoring, but I can't opt out or dismiss the dialog since the backend code is missing

**Comment by AndrewX192** [comment]
I was able to work around this with the following hack:
```
diff --git a/src/pages/create-account/steps/zip-code/zip-code.js b/src/pages/create-account/steps/zip-code/zip-code.js
index baf77f8..29799f4 100644
--- a/src/pages/create-account/steps/zip-code/zip-code.js
+++ b/src/pages/create-account/steps/zip-code/zip-code.js
@@ -62,6 +62,7 @@ export const ViewModel = StepViewModel.extend({
         const place = this.attr('place');
         // If we are auto-selecting Basic for them, we can skip zip code entry and
         // bypass to the address entry screen
+       return true;
         const preferredPlan = this.attr('place.base:tags')
           .filter(i => i.includes('PREF_PLAN:'));
         return (place.attr('place:state') && place.attr('place:zipCode'))
diff --git a/src/pages/promonitoring/promonitoring.js b/src/pages/promonitoring/promonitoring.js
index 364375d..6c7cf94 100644
--- a/src/pages/promonitoring/promonitoring.js
+++ b/src/pages/promonitoring/promonitoring.js
@@ -62,8 +62,8 @@ export const ViewModel = canMap.extend({
           ProMonitoringService.GetSettings(place.GetDestination()).then(({ settings }) => {
             this.attr('promonitoringSettings', settings);
             if (settings) {
-              const isPromon = this.attr('place.isPromon');
-              const isTrial = this.attr('promonitoringSettings').attr('promon:trial');
+              const isPromon = true; // this.attr('place.isPromon');
+              const isTrial = false; // this.attr('promonitoringSettings').attr('promon:trial');
               if (!isPromon && !isTrial) {
                 this.attr('promonitoringSettings').CheckAvailability().then(({ available }) => {
                   if (available !== 'FULL') {
(END)
```

I'll follow up once I have a better fix.

**Comment by AndrewX192** [comment]
I also found that I had to put my account in the "completed" status via oculus in order to get the prompt from showing up every time the page is loaded.

**Comment by zealgit** [comment]
I have clone successfully but not open url like http://localdev.irisbylowes.com:8443 how to run this project ?
please give me step.

**Comment by AndrewX192** [comment]
@zealgit I dumped my quickstart notes in #3, and I'll continue to add more details over the next few days. hope it helps. let's move the conversation there

**Comment by AndrewX192** [comment]
It appears that #12 didn't remove all the references (specifically a few imports) that cause distDocker to fail:

```
 ./gradlew :tools:khakis:distDocker
Starting a Gradle Daemon (subsequent builds will be faster)

> Task :installDeps

> i2web@2018.10.2 postinstall /home/wlnet/arcusweb
> npm-run-all bundle-dev


> i2web@2018.10.2 bundle-dev /home/wlnet/arcusweb
> steal-tools bundle --dev --no-minify

OPENING: /home/wlnet/arcusweb/package.json!npm
OPENING: i2web@2018.10.2#pages/create-account/create-account
WARN: unexpected closing tag </div>
WARN: unexpected closing tag </div> expected </p>
OPENING: i2web@2018.10.2#pages/invite/invite
OPENING: i2web@2018.10.2#pages/login/login
OPENING: i2web@2018.10.2#pages/home/home
WARN: expected closing tag </div>
OPENING: i2web@2018.10.2#pages/history/history
OPENING: i2web@2018.10.2#pages/scenes/scenes
OPENING: i2web@2018.10.2#pages/rules/rules
OPENING: i2web@2018.10.2#pages/devices/devices
OPENING: i2web@2018.10.2#pages/settings/service-plan/service-plan
ERROR: ENOENT: no such file or directory, open '/home/wlnet/arcusweb/src/components/settings/places/update-service-level/promon-to-basic.component'
ERROR: allDeps
Build failed

npm ERR! Linux 4.15.0-52-generic
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "run" "bundle-dev"
npm ERR! node v8.10.0
npm ERR! npm  v3.5.2
npm ERR! code ELIFECYCLE
npm ERR! i2web@2018.10.2 bundle-dev: `steal-tools bundle --dev --no-minify`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the i2web@2018.10.2 bundle-dev script 'steal-tools bundle --dev --no-minify'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the i2web package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     steal-tools bundle --dev --no-minify
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs i2web
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls i2web
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/wlnet/arcusweb/npm-debug.log
ERROR: "bundle-dev" exited with 1.

npm WARN optional Skipping failed optional dependency /chokidar/fsevents:
npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.2.9
npm WARN optional Skipping failed optional dependency /steal-tools/chokidar/fsevents:
npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.2.9
npm WARN i2web@2018.10.2 license should be a valid SPDX license expression
npm ERR! Linux 4.15.0-52-generic
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "install"
npm ERR! node v8.10.0
npm ERR! npm  v3.5.2
npm ERR! code ELIFECYCLE
npm ERR! i2web@2018.10.2 postinstall: `npm-run-all bundle-dev`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the i2web@2018.10.2 postinstall script 'npm-run-all bundle-dev'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the i2web package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm-run-all bundle-dev
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs i2web
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls i2web
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/wlnet/arcusweb/npm-debug.log

> Task :installDeps FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':installDeps'.
> Process 'command 'npm'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 5.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/4.10.2/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 1m 15s
1 actionable task: 1 executed
```

**Comment by AndrewX192** [comment]
This needs more work, but mostly covered at this point - going to leave open because there's still upgrade prompts after logging in

**Comment by AndrewX192** [comment]
These have been mostly fixed with #30
